packages/regl-worldview/package.json

```diff
@@ -33,5 +34,6 @@
   },
   "devDependencies": {
     "flow-bin": "^0.89.0"
-  }
+  },
+  "gitHead": "a799782ff908186f8c42c37f39806d251922ca6e"
```

Comment:
What's this for?

Reply:
Lerna added that automatically when publishing.